src/pages/reports/report-entity-listing/report-entity-listing.html

```diff
@@ -5,13 +5,18 @@
   Ionic pages and navigation.
 -->
 <ion-header>
-  <header [title]="'headings.'+ currentEntityType |translate" [hideBack]="false">
-  </header>
+  <header
+    [title]="'headings.'+ currentEntityType |translate"
```

Comment:
This you need not translate,, you can use currentEntityType directly

Reply:
ok